maps/away/generic/bigderelict.dm

```diff
@@ -2,7 +2,7 @@
 	name = "large derelict"
 	description = "A very large derelict station. According to the starmap, it shouldn't exist."
 	suffix = "generic/bigderelict.dmm"
-	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE)
+	sectors = list(SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_NEW_ANKARA, SECTOR_BADLANDS, SECTOR_AEMAQ)
```

Comment:
No reason to remove tau ceti and romanovich from the list.

Reply:
Think this was an accident on my part